Datagrid/Datagrid.php

```diff
@@ -126,15 +126,15 @@ public function buildPager()
             $this->formBuilder->add($filter->getFormName(), $type, $options);
         }
 
-        $this->formBuilder->add('_sort_by', 'hidden');
+        $this->formBuilder->add('_sort_by', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
```

Comment:
I couldn't make it work with symfony 2.7.11:

`Could not load type "Symfony\Component\Form\Extension\Core\Type\HiddenType"`

If I understand well, it would be 'hidden' instead of 'Symfony\Component\Form\Extension\Core\Type\HiddenType' and add a mapping in Sonata\AdminBundle\SonataAdminBundle for 3.x compatibility.